### PR TITLE
Replaced overall score's clear override div with a subtle button.

### PR DESCRIPTION
**Pull request by ihair**

### CODE REVIEW

File: d2l-rubric-overall-score.html [hunk 1]
```diff
@@ -305,6 +305,14 @@ <h4>
 				);
 			},
 
+			_clearOverride: function() {
```
Comment (mpharoah-d2l) [inline d2l-rubric-overall-score.html]: This shouldn't be necessary. The cell already has an action that will clear the override when clicked. You don't need to add another one to the button. (Or is the button preventing the click event from propagating?)

Comment (mpharoah-d2l) [reply d2l-rubric-overall-score.html]: Actually, do we really want this to be a button? The whole cell is clickable, not just the text, so a button doesn't really make sense

Comment (mpharoah-d2l) [reply d2l-rubric-overall-score.html]: Oh wait, this is overall score, not overall level. Nevermind, I was thinking of something else

Comment (ihair) [reply d2l-rubric-overall-score.html]: The designers want it to be a subtle button

**Pull request closed without merging**